execution/gethexec/arb_interface.go

```diff
@@ -18,10 +18,6 @@ type TransactionPublisher interface {
 	Start(context.Context) error
 	StopAndWait()
 	Started() bool
-
-	// This is only for testing the switch sequencer. Will be removed if the espresso light client
-	// contract is ready and we will use another way to trigger the mode switching.
-	SetMode(ctx context.Context, espresso bool) error
```

Comment:
Just for my own clarification, the removal of this function does not imply the deprecation of `SwitchToEspresso` and `SwitchToCentralized` for the purposes of integration tests correct?

Reply:
I am not quite clear what you meant. As you can see, the trigger to switch to another mode is our light client contract.
https://github.com/EspressoSystems/nitro-espresso-integration/blob/58578ae531da6f0a00ffb530c75fd36fe8213684/execution/gethexec/switch_sequencer.go#L117
Before the contract was done, we had finished the switching functionality and that was why we chose to add a manual trigger to test the `switch`.